app/game/memoryGame/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="script.js" defer></script>
+    <title>Memory Game</title>
+</head>
+<body>
```

Comment:
Let's put some message container here. We can swap out the text to let the player know what's going on.
`Memorize the pattern...`
`Repeat the pattern:`
`INCORRECT, try again!`
`Breach resolved!`